deny.toml

```diff
@@ -30,16 +30,32 @@ confidence-threshold = 0.8
 # List of explictly allowed licenses
 allow = [
   "Apache-2.0",
+  "BSD-2-Clause",
+  "CC0-1.0",
+  "ISC",
   "MIT",
   "MPL-2.0",
-  "ISC",
-  "BSD-2-Clause",
+  "OpenSSL",
+]
+[[licenses.clarify]]
+name = "ring"
+# SPDX considers OpenSSL to encompass both the OpenSSL and SSLeay licenses
+# https://spdx.org/licenses/OpenSSL.html
+# ISC - Both BoringSSL and ring use this for their new files
+# MIT - "Files in third_party/ have their own licenses, as described therein. The MIT
+# license, for third_party/fiat, which, unlike other third_party directories, is
+# compiled into non-test libraries, is included below."
+# OpenSSL - Obviously
+# Reference:  https://github.com/EmbarkStudios/cargo-deny/issues/110
+expression = "ISC AND MIT AND OpenSSL"
+license-files = [
+  {path = "LICENSE", hash = 0xbd0eed23},
 ]
 
 # https://embarkstudios.github.io/cargo-deny/checks/bans/cfg.html
 [bans]
 # Lint level for when multiple versions of the same crate are detected
-multiple-versions = "warn"
+multiple-versions = "deny"
```

Comment:
This one should probably be `warn` - we're aware there are multiple versions of some crates but unfortunately can't do anything about it right now.